sdk/formrecognizer/tests.yml

```diff
@@ -6,7 +6,7 @@ extends:
     ServiceDirectory: formrecognizer
     TimeoutInMinutes: 180
     CloudConfig:
-      Public:
+      Canary:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         Location: 'centraluseuap'
-    SupportedClouds: 'Public'
+    SupportedClouds: 'Canary'
```

Comment:
You'll need to add this:

```
    Clouds: 'Public'
```

SupportedClouds specifies all clouds, for which we only run the full set on the weekly pipeline. the `Clouds` parameter is what's read by the nightly live test pipeline, and defaults to `Public` under the hood.

Reply:
If that's the case, shouldn't we add:
```
    Clouds: 'Canary'
```
instead? We're trying to run in canary every day.

Reply:
Yep, my mistake.